### PR TITLE
[feat] Add Kafka WS and x509 services

### PR DESCRIPTION
**Pull request by eduardogmisiuk**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
Added Kafka WS and new EJBCA services. 

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:

### CODE REVIEW

File: deploy.yaml [hunk 1]
```diff
@@ -124,4 +113,8 @@
       tags: 100k
     - role: kafka2ftp
     - role: kafka-loopback
-      tags: [ never, 100k ]
+      tags: [never, 100k]
+    - role: kafka-ws
```
Comment (andersonluisribeiro) [inline deploy.yaml:117]: When you don't set a tag, kafka-ws should be deployed?

File: deploy.yaml [hunk 2]
```diff
@@ -124,4 +113,8 @@
...
+      tags: [100k]
+    - role: x509-identity-mgmt
+      tags: 100k
```
Comment (andersonluisribeiro) [inline deploy.yaml:120]: When you don't set a tag, x509-identity-mgmt should be deployed?

File: roles/apigw/templates/apigw_migration_job.yaml.j2 [hunk 3]
```diff
@@ -31,5 +31,5 @@ spec:
               name: kong-secret
               key: db
         image: dojot/kong:{{ dojot_apigw_version }}
-        args: ["kong", "migrations", "bootstrap"]
+        args: ["kong",  "migrations", "bootstrap"]
```
Comment (andersonluisribeiro) [inline roles/apigw/templates/apigw_migration_job.yaml.j2:34]: It might be better to remove unnecessary spaces

File: deploy.yaml (hunk 2 above)
Comment (eduardogmisiuk) [reply deploy.yaml:120]: Yes, it is the substitute for the old EJBCA.

File: deploy.yaml (hunk 1 above)
Comment (eduardogmisiuk) [reply deploy.yaml:117]: Yes.

**Pull request merged**